src/containers/Tenant/Schema/SchemaTree/SchemaTree.tsx

```diff
@@ -21,6 +24,9 @@ export function SchemaTree(props: SchemaTreeProps) {
 
     const dispatch = useDispatch();
 
+    const [_, setQueryMode] = useSetting<QueryMode>(QUERY_INITIAL_MODE_KEY);
+    const [enableAdditionalQueryModes] = useSetting<boolean>(ENABLE_ADDITIONAL_QUERY_MODES);
```

Comment:
what do you think about combining it into one hook, which will determine available modes and show an error toast if unavailable mode is going to be set.

Reply:
I'm not sure that it's currently needed. 
1) Is will be used only here, because in `QueryEditor` we will need both values to manage controls
2) It's a temporary solution while there is no ydb versions where QueryService is supported (it was supposed to be in 23-2, but 2 months ago we forgot to merge PR)

Reply:
I suggested this, because I don't like passing `enableAdditionalQueryModes` to `bindActions`. I think that the decision to change mode or not to change should be taken here, not in `bindActions`

Reply:
Now I've got your point. I'll change it